### PR TITLE
responsive menu bar

### PR DESCRIPTION
**Pull request by sooodi**
I added some images &  I forgot to cover the file navbar. I did them now

### CODE REVIEW

**Comment by lalomartins** [comment]
wth… where is the Cloudflare comment? 🤯 It build successfully to https://sudy-604-main-layout-and-men.socious-web-app.pages.dev/

File: .gitignore [hunk 1]
```diff
@@ -86,3 +86,4 @@ typechain/
 
 # If someone uses npm, make sure we don't commit the lock file, as it causes bugs
 frontend-next/package.json
+package-lock.json
```
Comment (lalomartins) [inline .gitignore:89]:
```suggestion
package-lock.json
```

File: frontend-next/.prettierrc [hunk 2]
```diff
@@ -1,8 +0,0 @@
-{
-  "printWidth": 80,
-  "singleQuote": true,
-  "trailingComma": "all",
-  "bracketSpacing": false,
-  "semi": true,
-  "arrowParens": "always"
-}
```
Comment (lalomartins) [inline frontend-next/.prettierrc:8]: You're deleting this but you didn't move the options to `prettier.config.js`, so the formatting for the whole repository changed

File: .gitignore (hunk 1 above)
Comment (lalomartins) [reply .gitignore:89]: also should add .DS_Store, not sure why it wasn't here already

**Review by lalomartins — CHANGES_REQUESTED** [review]
Under a certain width, an unneeded padding appears on the right side

https://user-images.githubusercontent.com/1211455/190151644-ff8c64e3-9660-423c-854a-a38370257b42.mp4

File: .gitignore (hunk 1 above)
Comment (sooodi) [reply .gitignore:89]: ok

Comment (sooodi) [reply .gitignore:89]: @lalomartins  please see my branch   .DS_Store existed

Comment (lalomartins) [reply .gitignore:89]: I mean it should be added to .gitignore. But that's ok, it doesn't need to be in this PR, I'll sort it out later

File: frontend-next/.prettierrc (hunk 2 above)
Comment (sooodi) [reply frontend-next/.prettierrc:8]: @lalomartins  done

**Comment by sooodi** [comment]
> wth… where is the Cloudflare comment? 🤯 It build successfully to https://sudy-604-main-layout-and-men.socious-web-app.pages.dev/

@lalomartins  that happend becuase  of merge .it is ok now

**Pull request closed without merging**